src/HttpCache/SouinPurger.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\HttpCache;
+
+use GuzzleHttp\ClientInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Purges Souin.
+ *
+ * @author Sylvain Combraque <darkweak@protonmail.com>
+ *
+ * @experimental
+ */
+class SouinPurger implements PurgerInterface
+{
+    private const MAX_HEADER_SIZE_PER_BATCH = 1500;
+    private const SEPARATOR = ', ';
+    private const SOUIN_COOKIE_NAME = 'souin-authorization-token';
+
+    private $logger;
+
+    // Clients to send cache invalidation
+    private $clients;
+
+    // Souin relative paths
+    private $dsn;
+    private $path;
+
+    // Souin token if mandatory
+    private $token;
+
+    /**
+     * @param ClientInterface[] $clients
+     */
+    public function __construct(
+        array $clients,
+        LoggerInterface $logger,
+        string $dsn,
+        string $path,
+        string $token
+    ) {
+        $this->clients = $clients;
+        $this->logger = $logger;
+        $this->dsn = $dsn;
+        $this->path = $path;
+        $this->token = $token;
+    }
+
+    private function getParametersFromIris(array $iris): string
+    {
+        return implode(self::SEPARATOR, $iris);
+    }
+
+    /**
+     * @param array|string[] $iris
+     *
+     * @return string[]
+     */
+    private function getChunkedRegex(array $iris): array
+    {
+        $regex = $this->getParametersFromIris($iris);
+        $batches = [];
+        $separatorLength = \strlen(self::SEPARATOR);
```

Comment:
Can't this be hardcoded (or stored in another private constant), as the parameter is a constant?